internal/migrations/20201019194303_change_overrides_to_text.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/openshift/assisted-service/models"
+	gormigrate "gopkg.in/gormigrate.v1"
+)
+
+const id = "20201019194303"
```

Comment:
Just realized that this will be a problem when we add a new migration as this is at the package level.